frontend/src/lib/components/LemonCalendar/LemonCalendarSelect.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { LemonCalendarSelect, LemonCalendarSelectProps } from 'lib/components/LemonCalendar/LemonCalendarSelect'
+import { Popup } from 'lib/components/Popup/Popup'
+import { LemonButton } from 'lib/components/LemonButton'
+import { dayjs } from 'lib/dayjs'
+import { formatDate } from 'lib/utils'
+
+export default {
+    title: 'Lemon UI/Lemon Calendar/Lemon Calendar Select',
```

Comment:
The "Lemon Calendar" folder in Storybook is pretty confusing with this layout:
It's 1 folder with 5 stories, but 3 pages and it's not intuitive you have to click "Lemon Calendar Range" and "Lemon Calendar Select" individually to see them:
<img width="239" alt="Screen Shot 2022-08-31 at 13 10 31" src="https://user-images.githubusercontent.com/4550621/187666120-cfbf3646-1c0f-4cf2-859d-f0fab7f81cfd.png">